docs/markdown/android/overlaypanel.md

```diff
@@ -0,0 +1,115 @@
+    Include Buttons in the header of the OverlayPanel as this can lead to sizing and internationalization problems.
+  </Group>
+</TwoCol>
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/06/e9/6f/06e96f0193da3a32a25d7f1174d5dfa1.png" alt="a partial overlay panel with a grabber to help resize"/>
+    <Do title="Do" />Utilize a grabber for Partial overlay panels that are able to be resized.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/62/a4/fa/62a4fa0befb32c836519b6d9f1af5f28.png" alt="a full overlay panel with rounded corners"/>
+    <Dont title="Don't" />
+    Round the corners of full overlay panels. Only Partial and Action overlay panels should have rounded corners.
+  </Group>
+</TwoCol>
+
+## Variants
+
+### Size
+
+OverlayPanels are available in three sizes.
+
+1. **Full overlay panel**
+   Full overlaypanel is always the full height of the viewport. They need to be dismissed to interact with underlying content.
+2. **Partial overlay panel**
+   Partial overlaypanel can be any height from 50% to 90% of the viewport. They are used when content on the overlaypanel relates to the layer behind, therefore a background wash is optional. They are dismissed by swiping down or tapping outside of the overlay panel area.
+3. **Action overlay panel**
+   Action overlay panels can be any height up to 50% of the viewport. They always include a background wash and need to be dismissed to interact with the underlying content. Action overlay panels can serve as an alternative to dialogs and inline menus.
+
+<ThreeCol>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/5a/47/1d/5a471d7d44d380915953f57777455c7d.png" alt="a Full overlay panel that takes up the full phone screen and shows a profile picture and the ability to see ideas from a pinner, along with other actions like downloading and changing the profile picture" />
+**Full overlay panel**
+</Group>
+
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/09/7a/72/097a72f92026753e50b29641eab975ff.png" alt="a Partial overlaypanel that can be closed and/or resized. It is for Creating a board" />
+**Partial overlay panel**
+</Group>
+
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/df/d3/8f/dfd38f800948c6bc8f3cb5a53d575a87.png" alt="An Action OverlayPanel that allows for sorting and organization" />
+**Action overlay panel**
+</Group>
+</ThreeCol>
+
+### Navigation
+
+OverlayPanels can utilize several navigation actions.
+
+1. **Close**
+   To close an overlay panel, use an icon button in the top left corner with the “close” icon.
+2. **Back**
+   To return to a previous screen, use an icon button in the top left corner with an “arrow-back” icon.
+
+3. **Button**
+   An inline Button can be used for a variety of purposes, back, close, next, save, etc.
+
+4. **Grabber**
+   A Grabber is a short gray line at the top of the overlay panel that tells the user that the overlay panel can be collapsed or expanded.
```

Comment:
"Grabber doesn't need to be capitalized here

Reply:
UNDOING